test/install_test.go

```diff
@@ -721,6 +736,9 @@ func TestCheckProxy(t *testing.T) {
 }
 
 func TestLogs(t *testing.T) {
+	// as of this point, don't fetch logs nor events if any test fails
+	fetchLogsEvents = false
```

Comment:
I don't really understand why we enable and disable this in a few places.  Is this to prevent infinite recursion or something?

Reply:
Before installing anything in the cluster, if a test fails, fetching the logs takes an eternity because it tries unsuccessfully to pull them for each container on each pod of the control plane, with a long timeout for each. So I'm only enabling fetching the logs after the tests that attempt installing the CP. And then I'm disabling that on the `TestLogs` test indeed for avoiding infinite recursion if that test fails.